src/general/ModLoader.cs

```diff
@@ -0,0 +1,320 @@
+        unloadedItemList = GetNode<ItemList>(UnloadedItemListPath);
+        loadedItemList = GetNode<ItemList>(LoadedItemListPath);
+
+        modInfoName = GetNode<Label>(ModInfoNamePath);
+        modInfoAuthor = GetNode<Label>(ModInfoAuthorPath);
+        modInfoVersion = GetNode<Label>(ModInfoVersionPath);
+        modInfoDescription = GetNode<Label>(ModInfoDescriptionPath);
+
+        confirmationPopup = GetNode<ConfirmationDialog>(ConfirmationPopupPath);
+
+        DirectoryInfo modFolder = Directory.CreateDirectory($"{Directory.GetCurrentDirectory()}\\mods");
+        foreach (DirectoryInfo currentMod in modFolder.EnumerateDirectories())
+        {
+            if (!File.Exists(currentMod.FullName + "/mod_info.json"))
+            {
+                continue;
+            }
+
+            var currentModInfo =
+                JsonConvert.DeserializeObject<ModInfo>(ReadJSONFile(currentMod.FullName + "/mod_info.json"));
+
+            if (currentModInfo.AutoLoad)
+            {
+                continue;
+            }
+
+            currentModInfo.Location = currentMod.FullName;
+            modList.Add(currentModInfo);
+            unloadedItemList.AddItem(currentModInfo.ModName);
+        }
+    }
+
+    // Copied From The PauseMenu.cs
+    private static string ReadJSONFile(string path)
+    {
+        using (var file = new Godot.File())
+        {
+            file.Open(path, Godot.File.ModeFlags.Read);
+            var result = file.GetAsText();
+
+            // This might be completely unnecessary
+            file.Close();
+
+            return result;
+        }
+    }
+
+    private void OnUnloadedModSelected(int index)
+    {
+        var tempModInfo = modList[index];
+        modInfoName.Text = tempModInfo.ModName;
+        modInfoAuthor.Text = "Author: " + tempModInfo.Author;
+        modInfoVersion.Text = "Version: " + tempModInfo.Version;
+        modInfoDescription.Text = tempModInfo.Description;
+        modInfoDescription.Text = tempModInfo.Description;
+        loadedItemList.UnselectAll();
+    }
+
+    private void OnLoadedModSelected(int index)
+    {
+        var tempModInfo = loadedModList[index];
+        modInfoName.Text = tempModInfo.ModName;
+        modInfoAuthor.Text = "Author: " + tempModInfo.Author;
+        modInfoVersion.Text = "Version: " + tempModInfo.Version;
+        modInfoDescription.Text = tempModInfo.Description;
+        modInfoDescription.Text = tempModInfo.Description;
+        unloadedItemList.UnselectAll();
+    }
+
+    private void OnMoveToLoadPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (unloadedItemList.GetSelectedItems().Length <= 0)
+        {
+            return;
+        }
+
+        var selectedItem = unloadedItemList.GetSelectedItems()[0];
+        loadedModList.Add(modList[selectedItem]);
+        loadedItemList.AddItem(modList[selectedItem].ModName);
+        modList.RemoveAt(selectedItem);
+        unloadedItemList.RemoveItem(selectedItem);
```

Comment:
You should add some extra blank lines here (and in other places where there are 4-5 or more lines without blanks in a row.